src/realm-roles/RoleAttributes.tsx

```diff
@@ -79,7 +87,7 @@ export const RoleAttributes = ({ form, save }: RoleAttributesProps) => {
                   <TextInput
                     name={`attributes[${rowIndex}].value`}
                     ref={form.register({
```

Comment:
you can remove this completely:

```suggestion
ref={form.register()}
```